server/src/metadata/field-metadata/field-metadata.auto-resolver-opts.ts

```diff
@@ -0,0 +1,44 @@
+import { SortDirection } from '@ptc-org/nestjs-query-core';
+import {
+  AutoResolverOpts,
+  PagingStrategies,
+  ReadResolverOpts,
+} from '@ptc-org/nestjs-query-graphql';
+
+import { JwtAuthGuard } from 'src/guards/jwt.auth.guard';
+
+import { FieldMetadata } from './field-metadata.entity';
+
+import { FieldMetadataService } from './services/field-metadata.service';
+import { CreateFieldInput } from './dtos/create-field.input';
+import { UpdateFieldInput } from './dtos/update-field.input';
+
+export const fieldMetadataAutoResolverOpts: AutoResolverOpts<
+  any,
+  any,
+  unknown,
+  unknown,
+  ReadResolverOpts<any>,
+  PagingStrategies
+>[] = [
+  {
+    EntityClass: FieldMetadata,
+    DTOClass: FieldMetadata,
+    CreateDTOClass: CreateFieldInput,
+    UpdateDTOClass: UpdateFieldInput,
+    ServiceClass: FieldMetadataService,
+    enableTotalCount: true,
+    pagingStrategy: PagingStrategies.CURSOR,
```

Comment:
Cursor pagination is the default strategy but I prefer to make things explicit, it also makes the config easier to edit if needed imho